src/cljs/athens/events.cljs

```diff
@@ -450,21 +450,36 @@
       {:fx [[:dispatch [:transact tx-data]]]})))
 
 
+(reg-event-fx
+  :page/reindex-left-sidebar
+  (fn [_ _]
+    {:doc "This is used in the `left-sidebar` to smooth out duplicate `:page/sidebar` values when bookmarked. "}
```

Comment:
Does this `:doc` show up anywhere?

Reply:
Just adding a docstring as metadata since `;;` was kinda ugly and this form didnt support `(defn foo "Prefered Doc-string" [x] )` notation.